src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -641,12 +696,13 @@ public void finalizeOutputFile(int testStatus) {
             log.error(e);
         }
         // Record the metrics
-        int passes = countInstancesOf("<td class='pass'>Pass</td>");
-        int fails = countInstancesOf("<td class='fail'>Fail</td>");
-        replaceInFile("STEPSPERFORMED", Integer.toString(fails + passes));
+        int passes = countInstancesOf("<td class='pass'>PASS</td>");
```

Comment:
What would be worlds better here is to not parse the HTML you've written out to the report already. Instead, you can store a LIst<Results> and run a foreach result.fail on it

Reply:
That is infinitely better. Again, I'll throw in a ticket to track it for future